### ISSUE TITLE
Character module issue

### ISSUE THREAD

**Issue by Siofral** [blish-hud/Blish-HUD#849]
I'm having a heck of a time with my GW2/Blish setup. Please tell me if you think this is GW2 or the module. I've been chasing my tail on this for a while.

If I click on the character icon the character panel comes up fully populated. If I try to switch to a different character I get the rolling choya bit, the chat input box is focused and nothing happens. The choya just keeps on a'rollin'.

To this I want to add one of my accounts cannot get a valid api token from ArenaNet. Two of my accounts have no issues with this so I know it's not a "me" thing.

I'm also using LaunchBuddy. All my accounts get launched without issue (I'm temporarily bailed on the alpha testing until I can find this bug) but Blish and Taco most often fail to load. Rather than fight it I've removed the paths in LB and am launching them individually as needed.

Any help/direction/ideas are greatly appreciated.

**Comment by Siofral** [comment]
In addition, when I log in (sans anything but BlishHud) I'm getting "ARCDps is unavailable"

**Comment by dlamkins** [comment]
The Guild Wars 2 API is currently experiencing problems, so you may have difficulties with some modules.

I'd need a screenshot of the ArcDPS item to comment much there, but our only interactions with ArcDPS are if you have the ArcDPS bridge added to ArcDPS, which is needed for certain modules.  Outside of that, we don't have any interactions.

**Comment by Siofral** [comment]
My problem is strictly with the GW2 API. I have two accounts with API keys Blish recognizes as valid. I have one account, my main account, where the API token, no matter how many times I delete and recreate, is ALWAYS deemed to be corrupt/invalid. I'm at my wit's end on this. Is this something I must wait on GW2 to resolve?

**Comment by dlamkins** [comment]
Yes, when the API is down there is nothing we can do. Certainly not if it fails to tell us if a token is valid. We just have to wait for the service to return. 

We have seen some accounts in the past experience issues with the API where tokens would fail to work until x amount of time after the game was purchased. I'm afraid we can't do anything about that either.

**Comment by Siofral** [comment]
I have managed to iron out this issue with ArenaNet Support. They have fixed the problem at their end which was a lesson unto itself. They initially said they would not support API issues but I played the squeeky wheel and the issue was resolved. I am blessed.
Now, however, I have a completely different issue. Where do I go to get help with the character module?
FYI, the character I'm selecting to have load is not loading. I am getting every other character on the account but NOT the one I've selected. ? Any ideas? Where do I go for help?
